react/components/Dropdown/index.js

```diff
@@ -110,7 +111,9 @@ class Dropdown extends Component {
     let iconSize
 
     let classes = 'bg-transparent bn w-100 h-100 '
-    let containerClasses = `${isInline ? '' : 'bw1'} br2 relative `
+    let containerClasses = `${styles.container} ${
+      isInline ? '' : 'bw1'
+    } br2 relative `
     let selectClasses = 'o-0 absolute top-0 left-0 h-100 w-100 bottom-00 '
 
     let labelClasses = 'vtex-dropdown__label db mb3 w-100 c-on-base '
```

Comment:
can u use `classNames`?

Reply:
didn't know I could! absolutely

Reply:
the idea is that if we are able to use classes, we will use `classNames`.

Reply:
btw, by `classNames` you mean [this package](https://www.npmjs.com/package/classnames), right?

Reply:
Yes!

Reply:
done!